scripts/runfabtests.sh

```diff
@@ -298,7 +295,7 @@ function main {
 	fi
 
 	for ts in ${tests}; do
-	ssh=${sssh}
+	ssh=${bssh}
```

Comment:
If we are dropping short/long variants we can just have assign `$ssh` at the top and skip this.